src/diffusers/schedulers/scheduling_euler_discrete.py

```diff
@@ -17,10 +17,11 @@
 from typing import List, Optional, Tuple, Union
 
 import numpy as np
+import scipy.stats
```

Comment:
so this I think would cause an error if user does not have scipy installed
1. can we look into if we can use beta sigmas without scipy?
2. if not, let's add 
```
if is_scipy_available:
    import scipy.stats
```

Reply:
I'd have to look into how feasible it is to reimplement `scipy.stats.beta.ppf`.

I've added the check though, thanks!